libnestutil/regula_falsi.h

```diff
@@ -30,16 +30,16 @@ namespace nest
 /**
  * Localize threshold crossing by using Illinois algorithm of regula falsi method.
  *
- * See https://en.wikipedia.org/wiki/Regula_falsi#The_Illinois_algorithm for details on the algorithm.
+ * See https://en.wikipedia.org/wiki/Regula_falsi#The_Illinois_algorithm for
+ * details on the algorithm.
  *
  * @param   Node   model object that call function
- * @param   function threshold_distance function
  * @param   double length of interval since previous event
  * @returns time from previous event to threshold crossing
  */
 template < typename CN >
 double
-regula_falsi( CN* node, double ( Node::*threshold_distance )( double ) const, const double dt )
+regula_falsi( CN* node, const double dt )
```

Comment:
I think `const CN& node` and a corresponding `regula_falsi(*this)` would be better.

Reply:
Done! :)